tests/test_llm_mock.py

```
@@ -0,0 +1,65 @@
+    )
+
+    # Verify that we get a mock result, not an actual LLM call
+    assert result is not None
+    # The result should have the attributes we expect from our mock
+    assert hasattr(result, "explicit")
+    assert hasattr(result, "deductive")
+    assert hasattr(result, "_response")
+
+
+@pytest.mark.asyncio
+async def test_summarizer_decorated_functions_with_mock():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_summarizer_decorated_functions_with_mock`

Add return type annotation: `None`

(ANN201)